### ISSUE TITLE
Style Tock Maintenance Page for Accessibility

### ISSUE THREAD

**Issue by carjug** [18F/tock#1535]
When we need to make production changes to the Tock Database, there is a maintenance app that gets deployed in place of Tock. Currently the app displays a page which is just plain (unstyled) HTML. We should fix this for accessibility reasons.

**Comment by Sgtpluck** [comment]
This should be timeboxed to be a small update to make the page screen-reader accessible, but not a full design style kind of thing.

**Comment by janelx** [comment]
Currently investigating acceptance criteria and if this duplicating work done in existing ticket, Accessibility/USWDS audit for compliance: https://github.com/18F/tock/issues/1542.

**Comment by janelx** [comment]
@mgwalker made changes to define language in header, and is waiting for review and merge.
